docs/topics/developers.md

```diff
@@ -81,6 +81,14 @@ $ eval $(minikube docker-env)
 Running `make docker-build docker-push` will push the Brigade images to the Minikube Docker
 daemon.
 
+From here, you can install Brigade into Minikube using the Helm chart:
+
+```
+$ helm install -n brigade ./charts/brigade
```

Comment:
How do you feel about the moment of being out of sync to images? happen often? or only now that it's early? Up to you.